src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(number) {
+      calculator.result += number;
+
+      return this;
+    },
+
+    subtract(number) {
+      calculator.result -= number;
+
+      return this;
+    },
+
+    multiply(number) {
+      calculator.result *= number;
+
+      return this;
+    },
+
+    divide(number) {
+      calculator.result /= number;
+
+      return this;
+    },
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+
+    operate(operation, number) {
+      operation(number);
```

Comment:
Look how you're always ignoring the callback return and assuming it will update `this.result`